### DISCUSSION TITLE
Card: South Lebanon Conflict

### DISCUSSION THREAD

**Discussion by RebelusQuo** [RebelusQuo/the-wall-is-down#15] [Ideas]
I think this card was missed when giving feedback in PRs https://github.com/RebelusQuo/the-wall-is-down/pull/1 and https://github.com/RebelusQuo/the-wall-is-down/pull/3 :

Clarification: What does this mean for US Prevalence? Do they automatically have Prevalence either always or with at least one
Influence present, or does it only stop other players so US still need to have more than any other?

**Comment by rodrigoSantamaria** [comment]
It only stops other players to have more influence than US in Lebanon. Since US cannot lose influence in Lebanon and gains 1 influence in Lebanon, normally it means that the US has automatic Prevalence by 1 (or more if later added) vs 0 influence of the rest. 
In the uncommon case that other players already had 2+ influence in Lebanon when this card is played (and US had less influence), then no player has Prevalence in Lebanon. Only US, by adding enough influence later on, can gain Prevalence in Lebanon, in that case.

**Comment by RebelusQuo** [comment]
Suggestion:
> Add 1 US Influence in Lebanon.
> **Continuous effect**: US cannot lose Influence in Lebanon, and no other player can Prevail there (US still needs majority for Prevalence).

**Comment by rodrigoSantamaria** [comment]
That is perfect
